grizzly_description/launch/description.launch

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2015, Clearpath Robotics
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Clearpath Robotics nor the
+      names of its contributors may be used to endorse or promote products
+      derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS BE LIABLE FOR ANY
+DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+This program is a launch file for the Clearpath Grizzly URDF
+-->
+<launch>
+  <arg name="accessories" default="false" />
+  <arg name="front_laser" default="false" />
+
+  <param name="robot_description"
+         command="$(find xacro)/xacro '$(find grizzly_description)/urdf/grizzly.urdf.xacro'
+                                      accessories:=$(arg accessories)
+                                      front_laser:=$(arg front_laser)
+                                      " />
```

Comment:
Jackal is env vars to turn things on and off in the description, and then that "env_run" wrapper to allow an arg interface for launching them in simulation:

http://wiki.ros.org/jackal_simulator#Advanced
https://github.com/jackal/jackal_simulator/blob/indigo-devel/jackal_gazebo/launch/jackal_world.launch#L28

But of course, if you don't like the pre-cooked "configs", you can still set env vars to get a more specific configuration. Or, once it's macro-ized like Husky, use the macro to spawn it into a larger URDF with whatever additional pieces defined in the surrounding URDF.